README.md

```diff
@@ -312,3 +312,62 @@ reported properly.
 
 *NOTE*: the current implementation does not handle all edge cases at this time.
 If you send invalid data, it is possible for it to silently fail.
+
+## Creating a fork on Heroku
```

Comment:
Since forking has many meanings, maybe a more specific title would be "How to fork this add-on in order to create your own Heroku add-on"